Dockerfile-api

```diff
@@ -6,15 +6,14 @@ COPY . /isaac-api
 RUN mvn package -Dmaven.test.skip=true
 
 # create clean jetty docker container
-FROM jetty:11.0.20-jdk17-eclipse-temurin AS server
+FROM jetty:12.0.6-jdk17-eclipse-temurin AS server
```

Comment:
It looks like there are different versions of Jetty (`12.0.6` and `12.0.8`) in different files (the 3 dockerfiles and the pom). We should keep all these versions the same. Ideally the latest one (`12.0.12`).